release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -101,6 +101,44 @@ type of private DNS zone.
 // Monitor whether this info moves to a KB article or remains in official docs.
 // If it moves to KB, permanently remove this entry.
 
+[id="ocp-4-5-machine-api"]
+=== Machine API
+
+[id="ocp-4-5-aws-machinesets-support-spot-instances"]
+==== AWS MachineSets support spot instances
+
+AWS MachineSets now support spot instances. This lets you create a MachineSet
+that deploys machines as spot instances so you can save costs compared to
+on-demand instance prices. You can configure spot instances by adding the
+following line under the `providerSpec` field in the MachineSet YAML file:
+
+[source,yaml]
+----
+providerSpec:
+  spotMarketOptions: 
+----
+
+[id="ocp-4-5-autoscaling-the-minimum-number-of-machines"]
+==== Autoscaling the minimum number of machines to 0
+
+For clusters running on AWS, Azure, and GCP, you can now set the minimum number
+of machines that must remain in the specified zone to `0`. This setting applies
```

Comment:
I think this needs clarification:
You can now set the minimum number of replicas for a `machineAutoscaler` to 0.
This enable the autoscaler to be even more cost effective by being able to scale from zero machines to anything and getting back to zero based on the resource your workloads require.